genTriposGAFFandGAFF2.py

```diff
@@ -92,38 +103,40 @@ def GenTriposGAFF(mol):
     os.chdir(opt.workdir)
 
     # make subdirectories for tripos and gaff files
-    make_path( 'tripos_mol2/' )
-    make_path( 'gaff_mol2/')
-
+    make_path('tripos_mol2/')
+    make_path('gaff_mol2/')
+    make_path('gaff2_mol2/')
     # read input file with all molecules
     ifs = oechem.oemolistream(inputfile)
-    
-    # Set up an error file to record skipped troublesome molecules
-    #errfile = open('errMols.txt', 'a')
 
+    ff = open("timer.dat","a")
+    delta=[]
     for mol in ifs.GetOEMols():
-
+        start = time.clock()
         try:
             tstart = time.clock()
             p = multiprocessing.Process(target=GenTriposGAFF, args=(mol,))
             p.start()
             p.join(25)
             if p.is_alive():
                 print("Skipping %s in %s due to timeout after 25 sec" % (mol.GetTitle(), inputfile))
-                #errfile.write("Skipping %s in %s due to timeout after 25 sec" % (mol.GetTitle(), inputfile))
                 p.terminate()
                 p.join()
         except Exception, e:
             print e, mol.GetTitle()
             continue
-
+        end = time.clock()
+        delta.append( end - start)
+    delta2=sum(delta)
+    # Set up a timer.dat file that keeps track of how long each molecule takein seconds
+    ff.write(str(inputfile)+('\t%.2f'%delta2)+'\n')
+    ff.close()
     # delete extra generated files
     commands.getoutput('rm ./gaff_mol2/*.frcmod')
```

Comment:
@bannanc, did you mention something about this no longer working in python3? 
If so, @dkyu, we might not need to change it for now, but we should make a note of it somewhere. (maybe a TODO section at the top of this script, or a TODO file in this github?)